### PR TITLE
Fixes for LB page and readonly blockspace

### PR DESCRIPTION
**Pull request by ajpal**
Relies on https://github.com/code-dot-org/blockly/pull/233

Before:
image

After (not perfect, but at least not broken):
image
 
Also switches the default on the pointer blocks to be the long string, so that it renders properly in instructions:
Before:
image

After:
image

## Links

- [jira](https://codedotorg.atlassian.net/browse/STAR-1169)

## Testing story

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/block_utils.js [hunk 1]
```diff
@@ -1040,7 +1039,11 @@ exports.createJsWrapperBlockCreator = function(
               });
             }
           });
-          if (appOptions.level.miniToolbox && !appOptions.readonlyWorkspace) {
+          if (
+            !window.appOptions ||
```
Comment (maddiedierker) [inline apps/src/block_utils.js:1046]: i don't understand the addition of `!window.appOptions` here -- why do we init mini-toolbox if we don't have appOptions?

(out of curiosity) are we moving away from using `/* global appOptions */` in favor of `window.appOptions`?

Comment (ajpal) [reply apps/src/block_utils.js:1046]: This is for the levelbuilder block page which doesn't have `appOptions` but we do want to render the mini toolbox blocks.

Comment (maddiedierker) [reply apps/src/block_utils.js:1046]: ah, right, that makes sense!

**Review by maddiedierker — APPROVED** [review]

File: apps/src/block_utils.js (hunk 1 above)
Comment (ajpal) [reply apps/src/block_utils.js:1046]: (added a comment to the code for clarity)

**Pull request merged**